metadata/src/test/java/org/apache/kafka/controller/MockAclControlManager.java

```diff
@@ -33,7 +33,7 @@
 public class MockAclControlManager extends AclControlManager {
     public MockAclControlManager(LogContext logContext,
                                  Optional<ClusterMetadataAuthorizer> authorizer) {
-        super(new SnapshotRegistry(logContext), authorizer);
+        super(new LogContext(), new SnapshotRegistry(logContext), authorizer);
```

Comment:
Seems like it should be `s/new LogContext()/logContext/`?

Reply:
fixed, thanks.